core/thread_invites.go

```diff
@@ -68,11 +70,13 @@ func (t *Thread) AddExternalInvite() (mh.Multihash, []byte, error) {
 	if err != nil {
 		return nil, nil, err
 	}
+	contact := t.datastore.Contacts().Get(t.node().Identity.Pretty())
 	msg := &pb.ThreadInvite{
 		Sk:        threadSk,
 		Name:      t.Name,
 		Schema:    t.schemaId,
 		Initiator: t.initiator,
+		Contact:   repoContactToProto(contact),
```

Comment:
This is needed because if you have never seen this contact before, you won't have their info. Also, we don't want to naively add it to the address book, so we just keep it on the indexed invite until it's been accepted.